src/calculateRentalCost.js

```diff
@@ -3,8 +3,28 @@
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
+
+  if (days >= 7) {
+    totalCost -= 50;
+  } else if (days >= 3) {
+    totalCost -= 20;
+  }
```

Comment:
Remove the `else` keyword since the `if` block contains a `return` statement, this simplifies the control flow as per checklist items 4 and 5.